torchvision/prototype/datasets/_builtin/coco.py

```diff
@@ -73,6 +95,62 @@ def resources(self, config: DatasetConfig) -> List[OnlineResource]:
         )
         return [images, meta]
 
+    def _segmentation_to_mask(self, segmentation: Any, *, is_crowd: bool, image_size: Tuple[int, int]) -> torch.Tensor:
+        from pycocotools import mask
+
+        if is_crowd:
+            segmentation = mask.frPyObjects(segmentation, *image_size)
+        else:
+            segmentation = mask.merge(mask.frPyObjects(segmentation, *image_size))
+
+        return torch.from_numpy(mask.decode(segmentation)).to(torch.bool)
+
+    def _decode_instances_anns(self, anns: List[Dict[str, Any]], image_meta: Dict[str, Any]) -> Dict[str, Any]:
+        image_size = (image_meta["height"], image_meta["width"])
+        labels = [ann["category_id"] for ann in anns]
+        categories = [self.info.categories[label] for label in labels]
+        return dict(
+            # TODO: create a segmentation feature
+            segmentations=torch.stack(
+                [
+                    self._segmentation_to_mask(ann["segmentation"], is_crowd=ann["iscrowd"], image_size=image_size)
+                    for ann in anns
+                ]
+            ),
+            areas=torch.tensor([ann["area"] for ann in anns]),
+            crowds=torch.tensor([ann["iscrowd"] for ann in anns], dtype=torch.bool),
+            bounding_boxes=BoundingBox(
```

Comment:
For a future PR, I think it might be preferable to rename the class to `BoundingBoxes` as we hold more than one box now.